.circleci/config.yml

```diff
@@ -5,43 +5,23 @@ commands:
       - run:
           name: Install Kustomize
           command: cd /usr/local/bin && curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | sudo bash
-  dep_ensure:
```

Comment:
I think you still want to cache dependencies. I've got this when I tried to migrate argocd to go modules:

```
  mod_download:
    steps:
      - restore_cache:
          keys:
            - go-mod-v1-{{ checksum "go.sum" }}
      - run:
          name: Run go mod download
          command: |
            [ -e "/home/circleci/.go_workspace/pkg/mod/$(sha1sum ./go.sum | cut -d " " -f 1)" ] || go mod download && touch "/home/circleci/.go_workspace/pkg/mod/$(sha1sum ./go.sum | cut -d " " -f 1)"
      - save_cache:
          key: go-mod-v1-{{ checksum "go.sum" }}
          paths:
            - /home/circleci/.go_workspace/pkg/mod
```

Reply:
good point

Reply:
done